src/locale/states/za.json

```
@@ -0,0 +1,11 @@
+{
+    "EC": "Eastern Cape",
+    "FS": "Free State",
+    "GP": "Gauteng",
+    "KZN": "Kwazulu-Natal",
```

According to https://www.gov.za/provinces, this is `KwaZulu-Natal`.  Should this be updated to match?

```suggestion
    "KZN": "KwaZulu-Natal",
```

---

Yes, that is the more common capitalization. One I'm giving is in ISO 3166.